pkgs/by-name/ne/needle/package.nix

```
@@ -0,0 +1,50 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, swift
+, swiftpm
+, swiftpm2nix
+, sqlite
+}:
+let
+  generated = swiftpm2nix.helpers ./nix;
+in
+stdenv.mkDerivation rec {
+  pname = "needle";
+  version = "0.24.0";
+
+  src = fetchFromGitHub {
+    owner = "uber";
+    repo = "needle";
+    rev = "v${version}";
+    hash = "sha256-vQlUcfIj+LHZ3R+XwSr9bBIjcZUWkW2k/wI6HF+sDPo=";
+  };
+
+  sourceRoot = "${src.name}/Generator";
+
+  nativeBuildInputs = [
+    swift
+    swiftpm
+  ];
+
+  propagatedBuildInputs = [ sqlite ];
+
+  configurePhase = generated.configure;
+
+  installPhase = ''
+    runHook preInstall
+    binPath="$(swiftpmBinPath)"
+    mkdir -p $out/bin
+    cp $binPath/needle $out/bin/
```

We can replace this with a one liner
```suggestion
    install -Dm755 "$(swiftpmBinPath)"/needle $out/bin/needle
```

---

Fixed, thanks!